### PR TITLE
Added $owner and $group parameters to customize ownership of config files and add support for Ubuntu

### PR DESCRIPTION
**Pull request by iuriaranda**
Two changes:

- Added $owner and $group parameters to customize ownership of config files
- Added support for Ubuntu

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -11,14 +11,12 @@
...
     }
-    'Debian': {
-      $sumo_package_name   = 'SumoCollector'
-      $sumo_service_config  = '/etc/sumo.conf'
-      $sumo_service_name  = 'collector'
-      $syncsources    = '/etc/sumo.sources.d'
-    }
-    'Ubuntu': {
-      $sumo_package_name   = 'sumocollector'
+    /^(Debian|Ubuntu)$/: {
```
Comment (ghoneycutt) [inline manifests/params.pp]: Ubuntu is not a valid `$osfamily`. The `$osfamily` for Ubuntu is `Debian`. 

The logic change here would work, though the case statement should be go back `'Debian': {`

**Review by ghoneycutt — CHANGES_REQUESTED** [review]

File: manifests/params.pp (hunk 1 above)
Comment (iuriaranda) [reply manifests/params.pp]: Fixed, that was a leftover.

File: manifests/conf.pp [hunk 2]
```diff
@@ -19,6 +19,8 @@
   $proxyuser       = undef,
   $sources         = undef,
   $syncsources     = undef,
+  $owner           = undef,
```
Comment (ghoneycutt) [inline manifests/conf.pp:22]: why are these set to undef? Shouldn't they stay root:root?

Comment (iuriaranda) [reply manifests/conf.pp:22]: This is to follow how the other parameters are defined. But ideally it should be `$::sumo::owner` and `$::sumo::group`, to get the default from the `sumo` class, but this should also be done for the rest of the parameters imo.

In any case, if not provided, puppet will default to root normally.

Comment (ghoneycutt) [reply manifests/conf.pp:22]: in order to not change the current behavior, please set the values to 'root'.

**Comment by iuriaranda** [comment]
Closing this for being legacy

**Pull request closed without merging**